### DISCUSSION TITLE
blog/paper-epilps-a-fast-and-flexible-bayesian-tool-for-estimation-of-the-time-varying-reproduction-number/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#99] [General]
# blog/paper-epilps-a-fast-and-flexible-bayesian-tool-for-estimation-of-the-time-varying-reproduction-number/

How to smooth epidemic curves and estimate the time-varying reproduction number in a flexible way? Read the EpiLPS methods paper (PLoS Computational Biology)

https://statsandr.com/blog/paper-epilps-a-fast-and-flexible-bayesian-tool-for-estimation-of-the-time-varying-reproduction-number/

**Comment by reghais** [comment]
Many thanks for sharing

**Reply by AntoineSoetewey** [reply]
You're welcome Azzeddine!
